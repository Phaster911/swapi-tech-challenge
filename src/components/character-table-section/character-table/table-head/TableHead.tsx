import styled from "styled-components";
import { QUERIES } from "../../../../style-constants";

const TableHead = styled.thead`
  display: none;

  @media ${QUERIES.tabletAndUp} {
    display: revert;
  }
`;

export default TableHead;
