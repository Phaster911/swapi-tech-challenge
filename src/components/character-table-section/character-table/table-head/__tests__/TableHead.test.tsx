import { render } from "@testing-library/react";
import TableHead from "../TableHead";

describe("<TableHead />", () => {
  it("renders the component", () => {
    const { container } = render(
      <table>
        <TableHead>
          <tr>
            <td>ABC</td>
          </tr>
        </TableHead>
      </table>
    );
    expect(container).toMatchSnapshot();
  });
});
