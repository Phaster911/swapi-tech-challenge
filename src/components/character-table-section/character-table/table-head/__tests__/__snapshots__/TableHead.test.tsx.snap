// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<TableHead /> > renders the component 1`] = `
.c0 {
  display: none;
}

@media (min-width: 750px) {
  .c0 {
    display: revert;
  }
}

<div>
  <table>
    <thead
      class="c0"
    >
      <tr>
        <td>
          ABC
        </td>
      </tr>
    </thead>
  </table>
</div>
`;
